app/mailers/audit_objects_mailer.rb

```diff
@@ -0,0 +1,15 @@
+# Send report of invalid objects to admin team
+class AuditObjectsMailer < ActionMailer::Base
```

Comment:
I've done this. I've only used it for the base of two of the mailers, since the others don't use the 'mailer' layout (I tested that if I set the layout to nil it works, but I would wonder if that would confuse things).